mmv1/third_party/terraform/services/netapp/resource_netapp_backup_vault_test.go

```diff
@@ -0,0 +1,109 @@
+package netapp_test
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+	"github.com/hashicorp/terraform-provider-google/google/tpgresource"
+	transport_tpg "github.com/hashicorp/terraform-provider-google/google/transport"
+)
+
+func TestAccNetappbackupVault_netappBackupVaultExample_update(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"random_suffix": acctest.RandString(t, 10),
+	}
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccNetappbackupVault_netappBackupVaultExample_basic(context),
+			},
+			{
+				ResourceName:            "google_netapp_backup_vault.test_backup_vault",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"location", "name", "labels", "terraform_labels"},
```

Comment:
This doesn't seem right. Why are we ignoring these values on import

Reply:
I followed the instructions at https://googlecloudplatform.github.io/magic-modules/develop/test/#add-an-update-test. The ImportStateVerifyIgnore line was auto-generated by magic modules. I just copied the *_generated_test.go file.
This looks like a google provider 4.x to 5.x change to me. The '*_generated_test.go' files of 4.x had less parameters in here, the 5.x ones now have these. With the old line, my tests were failing due to state issues, with the new ones not (for obvious reasons: they are ignored). AFAICT, that isn't a issue if my resource, but it is something in MM and google 5.x.